core/src/main/kotlin/io/github/serpro69/kfaker/provider/misc/RandomClassProvider.kt

```diff
@@ -155,7 +159,12 @@ class RandomClassProvider {
                 }
                 .toTypedArray()
 
-            constructor.call(*params)
+            try {
+                constructor.call(*params)
+            } catch (e: Exception) {
+                e.printStackTrace()
+                throw InstantiationException("Failed to instantiate $this with $params")
```

Comment:
This will only print "... with [Ljava.lang.Object;@35efd12c"

Can you move the `.toTypedArray()` from the variable into the call:

constructor.call(*params.toTypedArray())

Printing the list will always work.

Reply:
good point, I'll fix it

Reply:
Added in #185

Reply:
Added in #185